components/grid/templates/popup-form-template.md

```diff
@@ -11,13 +11,28 @@ position: 50
 
 # Popup Form Template
 
-With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the Grid. Declare the desired custom content inside the `<FormTemplate>` inner tag of the `<GridPopupEditFormSettings>`.
+With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the Grid. 
 
-You can use the `Context` attribute of the `<FormTemplate>` tag to set the name of the context variable. The context variable is of type `object` and can be cast to the model type to which the Grid is bound.
+>caption In this article:
+* [Basics](#basics)
+* [Specifics](#specifics)
+* [Example](#example)
 
->When using the template, the default Popup form is replaced by the declared content within the `FormTemplate` tag. Consequently, the default `Update` and `Cancel` buttons are removed. This means the [`OnUpdate` and `OnCancel`]({%slug components/grid/editing/overview%}#events) events cannot be triggered. To modify or cancel the update of a record, you need to include custom controls to manage these actions.
+## Basics
 
->caption Using a `FormTemplate` to modify the Edit/Create Popup window.
+1. Declare the desired custom content inside the `<FormTemplate>` inner tag of the `<GridPopupEditFormSettings>`. For example, [`TelerikForm`]({%slug form-overview%}) or [`EditForm`](https://learn.microsoft.com/en-us/aspnet/core/blazor/forms/?view=aspnetcore-7.0).
+1. The `FormTemplate` provides `context` of type `object`. Cast that to your model, so you can pass it to the custom form.
+1. (optional)Use the `Context` attribute of the `<FormTemplate>` tag to set the name of the context variable.
+
+## Specifics
+
+When using the template, the default Popup form is replaced by the declared content within the `FormTemplate` tag. This introduces the following specifics:
+* The default `Update` and `Cancel` buttons are removed. This means the [`OnUpdate` and `OnCancel`]({%slug components/grid/editing/overview%}#events) events cannot be triggered. To modify or cancel the update of a record, you need to include custom controls to manage these actions.
```

Comment:
UI element labels are [rendered in **bold**](https://docs.telerik.com/style-guide/element-formatting).
```suggestion
* The default **Update** and **Cancel** buttons are removed. This means that the [`OnUpdate` and `OnCancel`]({%slug components/grid/editing/overview%}#events) events cannot be triggered. To modify or cancel the update of a record, you need to include custom controls to manage these actions.
```